app/gui/view/src/documentation.rs

```diff
@@ -221,6 +233,23 @@ impl Model {
         self.inner_dom.dom().set_style_or_warn("padding", format!("{}px", padding));
         self.inner_dom.dom().set_style_or_warn("padding-top", format!("{}px", PADDING_TOP));
     }
+
+    /// Set the navigator so it can be disabled on hover.
+    pub fn set_navigator(&self, navigator: Option<Navigator>) {
```

Comment:
I don't understand this PR. No one is calling this function, how does it work?

Reply:
Oops, I missed one of the files with changes while committing. Fixed now.